stackslib/src/net/api/getblock_v3.rs

```diff
@@ -155,6 +155,10 @@ impl HttpRequest for RPCNakamotoBlockRequestHandler {
 
         Ok(HttpRequestContents::new().query_string(query))
     }
+
+    fn metrics_identifier(&self) -> &str {
+        "/v3/blocks/:block_id"
```

Comment:
The other `metrics_identifier()` strings use `:hash` instead of `:block_id`. I think `:block_id` is actually preferable because it's more specific, but we should be consistent, and either use `:block_id` or `:hash` everywhere

Reply:
Ah, ok. I chose `block_id` because it matches the regex above. I can add another commit to update the others.

Reply:
While I'm at it, I'll rearrange the method ordering of these new ones to match.

Reply:
7c6a3f0e8